### ISSUE TITLE
oci_cli/__init__.py Throwing ValueError when in dist-packages

### ISSUE THREAD

**Issue by sdstewar** [oracle/oci-cli#170]
https://github.com/oracle/oci-cli/blob/8d20e6e54805a792e9380f5dee52e6e90206d381/src/oci_cli/__init__.py#L30

The above line is throwing a ValueError when packages are pip installed into dist-packages.  Version 2.5.16 works fine.  

**Error**
```
root@synod-1804:/usr/local/lib/python2.7/dist-packages# oci
Traceback (most recent call last):
  File "/usr/local/bin/oci", line 7, in <module>
    from oci_cli.cli import cli
  File "/usr/local/lib/python2.7/dist-packages/oci_cli/__init__.py", line 30, in <module>
    python_cli_root_dir = this_file_path[0:this_file_path.index("/src/oci_cli")]
ValueError: substring not found
```

**Debug**
```
root@synod-1804:/usr/local/lib/python2.7/dist-packages# python -m pdb /usr/local/bin/oci
> /usr/local/bin/oci(4)<module>()
-> import re
(Pdb) b /usr/local/lib/python2.7/dist-packages/oci_cli/__init__.py:26
Breakpoint 1 at /usr/local/lib/python2.7/dist-packages/oci_cli/__init__.py:26
(Pdb) c
> /usr/local/lib/python2.7/dist-packages/oci_cli/__init__.py(26)<module>()
-> this_file_path = abspath(getsourcefile(lambda: 0))
(Pdb) n
> /usr/local/lib/python2.7/dist-packages/oci_cli/__init__.py(27)<module>()
-> if "site-packages" in this_file_path:
(Pdb) p this_file_path
'/usr/local/lib/python2.7/dist-packages/oci_cli/__init__.py'
(Pdb) n
> /usr/local/lib/python2.7/dist-packages/oci_cli/__init__.py(30)<module>()
-> python_cli_root_dir = this_file_path[0:this_file_path.index("/src/oci_cli")]
(Pdb) p this_file_path
'/usr/local/lib/python2.7/dist-packages/oci_cli/__init__.py'
```

**Comment by vish1** [comment]
Hi @sdstewar, Thanks for reporting this. Can you provide more details on your system (os, virtual environment) and installation process? We will get back with a fix soon.

**Comment by sdstewar** [comment]
Ubuntu 18.04 running in Parallels.  It was attempting the install as root using pip (installed via distro's python-pip package).

**Comment by vish1** [comment]
@sdstewar, This issue has been fixed in Version 2.5.22, Please test it out and let us know
